### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1]  Icon: Tooltip not read.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11246]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
NGX Core Shellbar(<fd-shellbar-actions> ). As per to meet accessibility requirement tooltip is needed for each icon so we have added tooltip 'Profile' for user under <fd-shellbar-actions>. Tooltip is visible in ui but still its not readable by screen reader
Code : <fd-shellbar-actions
    id="userDetails"
    [user]="userDetails"
    [userMenu]="userMenu"
    title="Profile"
    [closePopoverOnSelect]="true"
  >

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
For user icon add tooltip in NGX Core Shellbar(<fd-shellbar-actions>

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes
#### Did you search for similar issues?
Yes

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev  please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
I guess we would need to set `aria-label` on the user menu
